utf8/valid_default.go

```diff
@@ -0,0 +1,20 @@
+//go:build purego || !amd64
+// +build purego !amd64
+
+package utf8
+
+import (
+	segascii "github.com/segmentio/asm/ascii"
```

Comment:
I think we could go without renaming the package here as well

Reply:
Removed import aliases for this and the other similar imports in  https://github.com/segmentio/asm/pull/58/commits/aaba3a88ddf3d4c4b2fc06bdacc0d57bdb9604df